w3af/plugins/crawl/spider_man.py

```diff
@@ -185,8 +185,15 @@ def do_ALL(self):
             om.out.information(
                 'The user is navigating through the spider_man proxy.')
 
+        # convert relative URL to absolute if request came from CONNECT
+        if hasattr(self.server, 'chainedHandler'):
+            base_path = "https://" + self.server.chainedHandler.path
+            path = base_path + self.path
+        else:
+            path = self.path
+
         # Convert to url_object
-        path = URL(self.path)
+        path = URL(path)
```

Comment:
See my comments at #7484 It's completely separate issue -  I didn't realize the fix will get mixed together with the others, next time I'd probably should create a branch.